Javascript/searchStr.js

```diff
@@ -0,0 +1,28 @@
+/*
+문자 찾기
+한 개의 문자열을 입력받고, 특정 문자를 입력받아 해당 특정문자가 입력받은 문자열에 몇 개 존재하는지 알아내는 프로그램을 작성하세요.
+문자열의 길이는 100을 넘지 않습니다. 
+
+- 입력설명
+첫 줄에 문자열이 주어지고, 두 번째 줄에 문자가 주어진다. 
+- 출력설명
+첫 줄에 해당 문자의 개수를 출력한다. 
+-  입력예제 1
+COMPUTERPROGRAMMING 
+R
+- 출력예제 1 
+3
+*/
+const solution = (str, chracter) => {
+  let count = 0;
+  if (str.length > 100) return "최대 길이를 초과했습니다.";
```

Comment:
```suggestion
  if (str.length > 100) throw new Error("문자열의 길이는 100을 넘지 않습니다.");
```
리턴 타입이 number와 string이 섞여서 발생할 수 있는 것 보단 에러를 throw하는 게 버그 발생 가능성이 적을 것 같아요.

Reply:
👍 👍

Reply:
https://github.com/solbi9382/Algorithm_Study/pull/72/commits/757c1ed616273210a1f9afd5e1cd55c28024586c 에서 수정했습니다.